solr/core/src/java/org/apache/solr/handler/GraphHandler.java

```diff
@@ -92,24 +104,29 @@ public void inform(SolrCore core) {
     }
 
     // This pulls all the overrides and additions from the config
+    List<PluginInfo> pluginInfos = core.getSolrConfig().getPluginInfos(Expressible.class.getName());
+
+    // Check deprecated approach.
     Object functionMappingsObj = initArgs.get("streamFunctions");
     if(null != functionMappingsObj){
+      log.warn("solrconfig.xml: <streamFunctions> is deprecated for adding additional streaming functions to GraphHandler.");
       NamedList<?> functionMappings = (NamedList<?>)functionMappingsObj;
       for(Entry<String,?> functionMapping : functionMappings) {
         String key = functionMapping.getKey();
         PluginInfo pluginInfo = new PluginInfo(key, Collections.singletonMap("class", functionMapping.getValue()));
-
-        if (pluginInfo.pkgName == null) {
-          Class<? extends Expressible> clazz = core.getResourceLoader().findClass((String) functionMapping.getValue(),
-              Expressible.class);
-          streamFactory.withFunctionName(key, clazz);
-        } else {
-          StreamHandler.ExpressibleHolder holder = new StreamHandler.ExpressibleHolder(pluginInfo, core, SolrConfig.classVsSolrPluginInfo.get(Expressible.class));
-          streamFactory.withFunctionName(key, () -> holder.getClazz());
-        }
-
+        pluginInfos.add(pluginInfo);
       }
+    }
 
+    for (PluginInfo pluginInfo : pluginInfos) {
+      if (pluginInfo.pkgName != null) {
+        ExpressibleHolder holder = new ExpressibleHolder(pluginInfo, core, SolrConfig.classVsSolrPluginInfo.get(Expressible.class));
```

Comment:
This line is likely hiding a bug (and also a bug in StreamHandler) because `SolrConfig.classVsSolrPluginInfo.get(Expressible.class)` should always return null due to type mismatch on the map. Need a String key, but we also don't have any tests to prove this out. If you don't want to address that in this issue, feel free to file a new JIRA and tag me on it. :)

Reply:
@madrob can you push up a fix to this PR?

Reply:
> ... feel free to file a new JIRA ...

Opened https://issues.apache.org/jira/browse/SOLR-14212 for this.